src/util/badge.ts

```diff
@@ -63,7 +63,7 @@ export default class Badge {
       options += `(colour=${this.options.colour})`
     }
 
-    return `${this.name}: ${this.value} ${options}`
+    return `${this.name}: ${this.value}${options ? ' ' + options : ''}`
```

Comment:
Could maybe `.trim()` the string before returning instead of conditionally adding `options` 🤔